onnx/backend/base.py

```diff
@@ -83,11 +81,9 @@ def is_compatible(
 
     @classmethod
     def prepare(
-        cls, model: ModelProto, device: str = "CPU", **kwargs: Any  # noqa: ARG003
-    ) -> BackendRep | None:
-        # TODO Remove Optional from return type
+        cls, model: ModelProto, *args: Any, **kwargs: Any  # noqa: ARG003
```

Comment:
I don't see the benefit in doing that. We lose some information. device is not used but I tihnk it is easier to keep it if anybody wants to use this piece of code and create his own backend.

Reply:
Leaving unused parameters causes confusion. The demonstrations purpose should be assigned to examples.